shared/ClrFacade/Internal.cs

```diff
@@ -430,7 +430,12 @@ static object internalGetFieldOrProperty(Type t, string name, BindingFlags b, ob
       if (property == null)
          throw new ArgumentException($"Field or property name '{name}' not found on object of type '{t.Name}', for binding flags '{b.ToString()}'");
 
-      return property.GetValue(objOrNull, null);
+      var getFieldOrProperty = property.GetValue(objOrNull, null);
```

Comment:
This is something that was **automagically** converted for us by COM. Now, we'll just manually convert enums into int32 when using ```GetFieldOrProperty```.

Note that there's a specific test which checks for the opposite of this when a method returns an enum. The test method is called ```GetTestEnum``` and it explicitly checks the opposite. The method returns ```TestEnum``` and then there's a conversion to string and a string comparison is made.

If we decide that this makes no sense, and we should always convert enum to int then we can change how this is done. But for now, this matches the expectation of rClr.